code/modules/cargo/order.dm

```diff
@@ -101,7 +101,7 @@
 	var/requisition_text = "<h2>[station_name()] Supply Requisition</h2>"
 	requisition_text += "<hr/>"
 	requisition_text += "Order #[id]<br/>"
-	requisition_text+= "Time of Order: [station_time_timestamp()]<br/>"
+	requisition_text+= "Time of Order: [SSday_night.get_twentyfourhour_timestamp()]<br/>"
```

Comment:
This seems strictly worse in terms of names

Reply:
I updated it to be more readable, idk what else id name it since that's as informative as it gets